packages/storybook-vue/stories/3_components/badge/Badge.stories.mdx

```diff
@@ -0,0 +1,289 @@
+          Für nur 
+        </div>
+        <div style="text-align:center; font-size: 60px; font-family: TeleNeo; color: white; font-weight: 700;">
+          <span>
+            15 €
+          </span>
+        </div>
+      </div>
+    </scale-badge>
+  `,
+});
+
+export const Template2 = (args, { argTypes }) => ({
+  components: { ScaleBadge },
+  props: {
+    label: String,
+    ...ScaleBadge.props,
+  },
+  template: `
+      <scale-badge :size="size" :color="color" :rotation="rotation">
+      <div>
+        <div style="text-align: center; font-size: 15px; font-family: TeleNeo; color: white; font-weight: 400; margin-bottom: 10px;">
+          Für nur
+        </div>
+        <div style="text-align: center; font-size: 40px; font-family: TeleNeo; color: white; font-weight: 700;">
+          <span>
+            15 €
+          </span>
+        </div>
+      </div>
+    </scale-badge>
+  `,
+});
+
+export const Template3 = (args, { argTypes }) => ({
+  components: { ScaleBadge },
+  props: {
+    label: String,
+    ...ScaleBadge.props,
+  },
+  template: `
+      <scale-badge :size="size" :color="color" :rotation="rotation">
+      <div>
+        <div style="text-align: center; font-size: 25px; font-family: TeleNeo; color: white; font-weight: 400; margin-bottom: 10px;">
+          ab
+        </div>
+        <div style="text-align: center; font-size: 35px; font-family: TeleNeo; color: white; font-weight: 700;">
+          <span>
+            19,95 €
+          </span>
+          <sup title="This offer is only valid for a limited time. Free shipping on all orders." style="font-size: 30px; color: white; cursor: help;">*</sup>
+        </div>
+      </div>
+    </scale-badge>
+  `,
+});
+
+
+## Standard
+### All default values
+
+<Canvas withSource="none">
+  <Story
+    name="Standard"
+    args={{
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+<ArgsTable story="Standard" />
+
+```html
+<scale-badge>
+  <div>
+    <div
+      style="text-align:center; font-size:20px; font-family:TeleNeo; color:white; font-weight:400;"
+    >
+      Für nur
+    </div>
+    <div
+      style="text-align:center; font-size: 60px; font-family: TeleNeo; color: white; font-weight: 700;"
+    >
+      <span> 15 € </span>
+    </div>
+  </div>
+</scale-badge>
+```
+
+
+
+### Scoped CSS variables
+
+```css
+ :host {
+  --color-badge:#e20074;
+  --inner-width-small: 86px;
+  --inner-height-small: 88px;
+  --inner-width-big: 126.5px;
+  --inner-height-big: 96px;
+  --width-small: 120px;
+  --height-small: 120px;
+  --height-big: 160px;
+  --width-big: 160px;
+}
+```
+
+
+
+
+
+
+
+## Size
+There are two sizes: 
+big / small. Big has the size 160x160 and small 120x120. In the example below, the badge is small
+
+<Canvas withSource="none">
+  <Story
+    name="Sizes"
+    args={{
+      size: "small",
+      color: "black",
+    }}
+  >
+    {Template2.bind({})}
+  </Story>
+</Canvas>
+
+```html
+<scale-badge style="--badge-color:blue" size:"small" >
```

Comment:
the style attribute does nothing, right? let's remove it to avoid confusing readers